app/services/cycle_timetable.rb

```diff
@@ -66,11 +66,16 @@ class CycleTimetable
     },
     2024 => {
       find_opens: Time.zone.local(2023, 10, 3, 9), # First Tuesday of October
-      apply_opens: Time.zone.local(2022, 10, 10, 9), # Second Tuesday of October
-      apply_1_deadline: Time.zone.local(2023, 9, 5, 18), # 1st Tuesday of September
-      apply_2_deadline: Time.zone.local(2023, 9, 19, 18), # 2 weeks after Apply 1 deadline
-      reject_by_default: Time.zone.local(2023, 9, 27, 23, 59, 59), # 1 week and a day after Apply 2 deadline
-      find_closes: Time.zone.local(2023, 10, 2, 23, 59, 59), # The evening before the find opens in the new cycle
+      apply_opens: Time.zone.local(2023, 10, 10, 9), # Second Tuesday of October
+      show_summer_recruitment_banner: Time.zone.local(2024, 7, 1), # To be defined the dates for this banner
+      apply_1_deadline: Time.zone.local(2024, 9, 5, 18), # 1st Tuesday of September
```

Comment:
According to [Dates and deadlines](https://www.gov.uk/government/publications/recruiting-postgraduate-trainee-teachers-recruitment-cycle-dates/recruiting-postgraduate-trainee-teachers-recruitment-cycle-dates) this should be 3 September 2024 at 6pm?

```suggestion
      apply_1_deadline: Time.zone.local(2024, 9, 3, 18), # 1st Tuesday of September
```